### PR TITLE
Add scroll bar when more layers added

### PR DESCRIPTION
**Pull request by junqiu-lei**
### Description
Add scroll bar when more layers added on panel, set a max height to the layer list panel. In screen displayed layers will be maximum to 6.

image

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._ 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: public/components/layer_control_panel/layer_control_panel.scss [hunk 1]
```diff
@@ -22,6 +22,12 @@
   .euiListGroupItem__label {
     width: $euiSizeL * 6;
   }
+
+  .euiDroppable {
+    overflow-y: auto;
+    overflow-x: hidden;
+    max-height: $euiSizeL * 8;
```
Comment (naveentatikonda) [inline public/components/layer_control_panel/layer_control_panel.scss:29]: What is the default value of this elastic ui variable $euiSizeL?

Comment (junqiu-lei) [reply public/components/layer_control_panel/layer_control_panel.scss:29]: Per euiSizeL is 24 px, so here max-height is 192 px. Per layer is 32 px height, maximum in screen displayed  layers number is 6.

**Comment by junqiu-lei** [comment]
The unit test and e2e failed due to https://github.com/opensearch-project/OpenSearch-Dashboards/commit/de89c527dee3a997bfffc2079dbf97e4f813e54e#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R136, also found in other plugins, asked @zhongnansu for helping look into it

**Comment by VijayanB** [comment]
Did you check moving layers still works?

**Comment by junqiu-lei** [comment]
> Did you check moving layers still works?

Yes, it works

**Review by VijayanB — APPROVED** [review]

**Review by naveentatikonda — APPROVED** [review]

**Pull request merged**